imagetagger/imagetagger/images/templates/images/download.py

```diff
@@ -1,154 +1,219 @@
+    export_format_data['export_format_id'] = export_id
+
+    response = requests.post(BASE_URL + "annotations/api/export/create/", data=export_format_data,
+                             cookies=login_data[1], headers={'referer': BASE_URL})
+    if response.status_code == 403:
+        print(f'You cannot create exports for dataset {set_id}.')
+        return False
+
+    response = response.json()
+    export_url = BASE_URL + f"/annotations/export/{response['export_id']}/download/"
+    r = requests.get(export_url,
+                     data=login_data[0],
+                     cookies=login_data[1],
+                     allow_redirects=False,
+                     headers={'referer': BASE_URL},
+                     stream=True)
 
+    export_name = os.path.join(target_folder, str(set_id) + ".txt")
+    with open(export_name, "wb") as f:
+        f.write(r.content)
+    print(f'Downloaded annotations for imageset {set_id}')
+    return True
 
-for imgset in imagesets:
-    if imgset != " ":
-        if not separate_download:
-            download_zip(imgset)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='This script will download images from the specified imageset for you.'
+                                                 f' The images will be downloaded from: {BASE_URL}')
+    parser.add_argument('-s', '--separate', action='store_true', default=False,
+                        help='Download the images separately instead of bundled in a ZIP file')
+    parser.add_argument('-a', '--annotations', action='store_true', default=False,
+                        help='Also download annotations')
+    parser.add_argument('-e', '--export-format', type=int,
+                        help='Export format to use in the annotation download. '
+                             f'Please visit {BASE_URL}/annotations/api/export_format/list/ while logged in '
+                             'to find the id of your export format')
+    parser.add_argument('-u', '--username', type=str, help='ImageTagger username')
+    parser.add_argument('-d', '--directory', type=str, default=os.getcwd(),
+                        help='Folder where imagesets will be stored in subdirectories'
+                             'named after their id. Default is the current directory')
+    parser.add_argument('imagesets', nargs='+', type=int, metavar='SET_ID',
+                        help='A list of imagesets to download')
+    args = parser.parse_args()
+
+    if not args.username:
+        args.username = input("Username: ")
+
+    password = getpass.getpass()
+
+    if args.annotations and not args.export_format:
+        print(f'Please visit {BASE_URL}/annotations/api/export_format/list/ '
+              'while logged in to find the id of your export format')
+        format_id = input("Enter your export format id: ")
+        if not format_id.isdigit():
+            sys.exit(f'{format_id} is not a valid integer.')
+        args.export_format = format_id
```

Comment:
```suggestion
        format_id = input("Enter your export format id: ")
        while not format_id.isdigit():
            print(f'{format_id} is not a valid integer.')
            format_id = input("Enter your export format id: ")
        args.export_format = format_id
```
This would allow fixing a typo without having to enter username and password again